test/conftest.py

```diff
@@ -313,6 +313,13 @@ async def AppList(self, stream):
             apps.append(api_pb2.AppStats(name=app_name, description=app_name, app_id=app_id))
         await stream.send_message(api_pb2.AppListResponse(apps=apps))
 
+    async def AppStop(self, stream):
```

Comment:
Currently `modal dict delete` is implemented by hitting the `AppStop` method — is there a better way to do that?

Reply:
I see, that's not great because it ties the client to the "dummy app data model" we've been trying to get away from (it's a long journey because we need to wait for old clients to be fully deprecated before we can actually change the data model server-side).

Take a look at the `Volume` class which has a `VolumeDelete` endpoint